utils.py

```diff
@@ -80,17 +80,18 @@ def max_power2_divisor(number: int, limit: int = 4096) -> int:
     return limit
 
 
-def get_vlan_tags_and_masks(start: int, end: int) -> list[tuple[int, int]]:
+def get_vlan_tags_and_masks(tag_ranges: list[list[int]]) -> list[str]:
```

Comment:
The typing hints make it completely clear, but the docstring has become obsolete if you could update it that now it works with a list of ranges (tag ranges)

Reply:
Updated docstrings